docs/docs/how-tos/connect-via-ssh.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: connect-via-ssh
+title: Connect to Nebari via SSH or SFTP
+description: Set up a remote SSH connection using jupyter-ssh
+---
+
+# Connect to Nebari via SSH or SFTP
+
+Secure Shell Protocol (SSH) is a common protocol for communicating securely over a network connection. Secure File Transfer Protocol (SFTP) is based on SSH and is used to transfer files to and from a remote machine. In this guide, you will learn how to connect to your Nebari deployment via SSH, and how to transfer files using SFTP from your local machine. 
+
+Nebari includes a package called [`jupyterhub-ssh`](https://github.com/yuvipanda/jupyterhub-ssh) in all deployments to allow remote connection to Nebari deployments. On Nebari, SSH connections are set up on port 8022 and SFTP connections on port 8023. 
+
+## Prepare for the connection on the Nebari deployment
+
+Nebari, at its core, is running on a JupyterHub server. You will create a JupyterHub _token_ in order to access your server remotely. To do this, log into your Nebari deployment and navigate to the JupyterHub token page, {your-nebari-domain.org/hub/token}. 
```

Comment:
It'd be nice to include how to navigate via the GUI - 
![image](https://github.com/user-attachments/assets/22c0df86-231e-4dc0-ba00-20f97c3443c5)

Reply:
The curly brackets seem strange to me.  I would just use normal parenthesis.